_component_design/assisted-search.md

```diff
@@ -0,0 +1,120 @@
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading state
+
+If autocomplete is taking time to load, a loading spinner should be used in place of results.
+
+User should still have instructions for how to search without autocompleted results (Enter).
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assist search loading state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-loading.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Search results
+
+Search completion results should be divided up by categories.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search - search results specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-results.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Assisted search states
+
+**Hover:** The user may click or use their arrow keys to select one of the items in the dropdown list.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search results specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-bar/search-assisted-states-1.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+**Active:** When a keyword is actively being used but has not yet been completed with search criteria.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search results - active state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-bar/search-assisted-states-2.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- **Filter + No Autocomplete:** Keyed entries that have not formed a complete query string should not have a background color.
+
+- **Complete:** When a filter+search pill has been completed. 
+
+- **Delete:** Backspace should delete a single character and not the entire query and/or search pill.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Search Pills for the Active and Complete states are documented as a variation of the Search Bar and Assisted Search bar component."] [class:"image bg-light border"] %}
```

Comment:
remove caps

Reply:
This doesn't make sense either (at least, read very well)... "that are documented" - is that what you mean?

Reply:
Not sure, I rewrote to:

"Search pills are used to display the active and complete states in the assisted search pattern"